### PR TITLE
feat(RowV2): new component

### PR DESCRIPTION
**Pull request by fabienhebert**
## Type

- Feature

### Summarise concisely:

A utility/display component to simplify our usage :
- We can replace Grid/Row/Col, by using multiples RowV2 (check story example)
- We can replace Stack with styled children, when each children should take a specific width

The component is based on `display:grid` and  templateColumn receive a `grid-template-column` value

### CODE REVIEW

File: packages/ui/src/components/RowV2/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+type StyledRowProps = Pick<RowV2Props, 'gap' | 'templateColumns' | 'alignItems'>
+export const StyledRow = styled('div', {
+  shouldForwardProp: prop =>
+    !['templateColumns', 'gap', 'alignItems'].includes(prop),
+})<StyledRowProps>`
+  display: grid;
+  ${({ theme, gap = 2, alignItems = 'normal', templateColumns }) => `
+    grid-template-columns: ${templateColumns};
+    grid-gap: ${theme.space[gap]};
```
Comment (chambo-e) [inline packages/ui/src/components/RowV2/index.tsx:14]:
```suggestion
    gap: ${theme.space[gap]};
```

File: packages/ui/src/components/RowV2/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  id?: string
+  className?: string
+  dataTestId?: string
+  children: ReactNode
+  templateColumns: string
+  gap?: keyof SCWUITheme['space']
+  alignItems?: CSSProperties['alignItems']
+}
+
+export const RowV2 = forwardRef(
```
Comment (chambo-e) [inline packages/ui/src/components/RowV2/index.tsx:29]: Why the forwardRef ? 🤔

File: packages/ui/src/components/RowV2/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    grid-template-columns: ${templateColumns};
+    grid-gap: ${theme.space[gap]};
+    align-items: ${alignItems};
+  `}
+`
+
+type RowV2Props = {
+  id?: string
+  className?: string
+  dataTestId?: string
```
Comment (chambo-e) [inline packages/ui/src/components/RowV2/index.tsx:22]: Why not use the native name ?

File: packages/ui/src/components/RowV2/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import { forwardRef } from 'react'
+import type { SCWUITheme } from '../../theme'
+
+type StyledRowProps = Pick<RowV2Props, 'gap' | 'templateColumns' | 'alignItems'>
+export const StyledRow = styled('div', {
+  shouldForwardProp: prop =>
+    !['templateColumns', 'gap', 'alignItems'].includes(prop),
+})<StyledRowProps>`
+  display: grid;
+  ${({ theme, gap = 2, alignItems = 'normal', templateColumns }) => `
```
Comment (chambo-e) [inline packages/ui/src/components/RowV2/index.tsx:12]: Why the default gap here ?

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/RowV2/index.tsx (hunk 2 above)
Comment (fabienhebert) [reply packages/ui/src/components/RowV2/index.tsx:29]: I think it's better to have it on every component (we think about that). It's optional but let say you wanna scroll to this row for some reason you can it's convenient and there are multiples use case like that.

File: packages/ui/src/components/RowV2/index.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+`
+
+type RowV2Props = {
+  id?: string
+  className?: string
+  dataTestId?: string
+  children: ReactNode
+  templateColumns: string
+  gap?: keyof SCWUITheme['space']
+  alignItems?: CSSProperties['alignItems']
```
Comment (matthprost) [inline packages/ui/src/components/RowV2/index.tsx:26]: no `justifyContent`?

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by chambo-e — APPROVED** [review]

**Review by matthprost — CHANGES_REQUESTED** [review]
Documentation is unreadable in dark theme

Screenshot 2023-03-17 at 15 35 19

**Review by matthprost — APPROVED** [review]

**Pull request merged**